spec/features/news_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+feature 'News' do
+  describe 'autirized as user' do
+    let(:user) { create :user }
+    let!(:news) { create :news, user_id: user.id }
+    let(:title) { attributes_for(:news)['title'] }
+
+    before(:each) do
+      sign_in user
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+    end
+
+    scenario 'moves to news list page' do
```

Comment:
Try

We should follow DRY principle, so try use describe, context, before block.

```ruby
  describe 'news list page' do
    before { visit news_index_path }

    scenario 'contains news' do
      expect(page).to have_content(title)
    end

    scenario 'does not contain add button' do
      expect(page).not_to have_button('Add news')
    end

    scenario 'moves to chosen news page' do
      click_link news.title

      expect(page).to have_content(title)
    end
  end

  describe 'news page' do
    before { visit news_path(news.id) }

    context 'does not contain' do
      scenario 'edit button' do
        expect(page).not_to have_button('Edit')
      end

      scenario 'delete button' do
        expect(page).not_to have_button('Delete')
      end
    end
  end